ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResource.java

```diff
@@ -97,4 +81,27 @@ public Response streamQuery(KsqlRequest request) throws Exception {
       ));
     }
   }
+
+  private TopicStreamWriter getTopicStreamWriter(Map<String, Object> clientLocalProperties, PrintTopic statement) {
+    PrintTopic printTopic = statement;
+    String topicName = printTopic.getTopic().toString();
```

Comment:
Is this right? the topic name would be "topic something". Maybe add a `topicName` method to `PrintTopic`?

Reply:
printTopic.toString() would give you ' topic stuff' - getTopic returns QualifiedTopic - which implements  toString with correct formatting. I removed the redundant reference on the line below